src/main/java/org/junit/runners/InvalidTestClassError.java

```diff
@@ -0,0 +1,43 @@
+package org.junit.runners;
```

Comment:
I think this class should be in the `org.junit.runners.model` package (with `InitializationError` etc.).